frontend/src/pages/_app.tsx

```diff
@@ -121,6 +121,7 @@ function App({ Component, pageProps }: AppPropsWithLayout): JSX.Element {
             <ThemeProvider theme={theme}>
               <Layout>
                 <Component {...pageProps} />
+                <div id="bp-toaster"></div>
```

Comment:
this is needed for Blueprint's toaster to work properly in React 18 - see PR description for details